xcodeproj/internal/xcodeproj_macro.bzl

```diff
@@ -20,12 +20,36 @@ def xcodeproj(*, name, xcodeproj_rule = _xcodeproj, schemes = None, **kwargs):
 
     project = kwargs.get("project_name", name)
 
+    focused_targets = [
+        bazel_labels.normalize(t)
+        for t in kwargs.pop("focused_targets", [])
+    ]
+    unfocused_targets = [
+        bazel_labels.normalize(t)
+        for t in kwargs.pop("unfocused_targets", [])
+    ]
+
+    if focused_targets and unfocused_targets:
+        fail("""\
+Only one of `focused_targets` or `unfocused_targets` can be set.
+""")
```

Comment:
I guess we can support both? I'll make it so `unfocused_targets` takes precedence (will remove things first).